webextensions/api/webNavigation.json

```diff
@@ -523,7 +523,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "≤58"
+                  "version_added": "22"
```

Comment:
✅ https://github.com/chromium/chromium/commit/271b2a8f23087882623b480843970e1eb0804bc8 in [22.0.1212.0](https://github.com/chromium/chromium/releases/tag/22.0.1212.0) before 22.0.1229.